.gitignore

```diff
@@ -18,6 +18,7 @@
 .env
 node_modules
 npm-debug.log
+client/npm-debug.log*
```

Comment:
Thanks! 

I just realized npm-debug.log* will suffice. All I wanted to do is ignore those timestamped npm-debug.log.201501...etc  files